ctru-rs/examples/gfx-3d-mode.rs

```diff
@@ -61,9 +61,8 @@ fn main() {
             buf.copy_from(IMAGE.as_ptr(), IMAGE.len());
         }
 
-        // Flush and swap framebuffers
-        gfx.flush_buffers();
-        gfx.swap_buffers();
+        left.flush_buffer();
+        left.swap_buffers();
```

Comment:
this probably should be flushing whichever side is being drawn to, not just the left screen?

Reply:
The issue is with the "stereo" parameter. Since the screen is in 3D mode, and the `TopRight` screen returns "top" as its raw value, the swap/flush would be done twice on the top screen, once when "Flushing left" and once when "flushing right" (it flushes both of them each time). I guess that means this function should live in `TopScreen3D`, but that doesn't implement `Screen`, so it's a bit of an issue.